packages/spatial/src/input/systems/ClientInputSystem.tsx

```diff
@@ -132,7 +133,13 @@ const worldPosInputComponent = new Vector3()
 const inputXRUIs = defineQuery([InputComponent, VisibleComponent, XRUIComponent])
 const inputBoundingBoxes = defineQuery([InputComponent, VisibleComponent, BoundingBoxComponent])
 const inputObjects = defineQuery([InputComponent, VisibleComponent, GroupComponent])
-const spatialInputObjects = defineQuery([InputComponent, VisibleComponent, TransformComponent, Not(CameraComponent)]) //TODO may be overkill if visible means it always has transform
+const spatialInputObjects = defineQuery([
+  InputComponent,
+  VisibleComponent,
+  TransformComponent,
+  Not(CameraComponent),
+  Not(XRScenePlacementComponent)
```

Comment:
ideally in the future we will figure out how to not have to specify many opt-outs here, but it's acceptable for now